app/serializers/spree/v2/storefront/vendor_serializer.rb

```diff
@@ -4,7 +4,10 @@ module Storefront
       class VendorSerializer < BaseSerializer
         set_type :vendor
 
-        attributes :name, :about_us
+        attributes :name, :about_us, :slug, :state, :contact_us
+
+        has_one :image
```

Comment:
@poudelprakash we need to declare a serializer for this

Reply:
@damianlegawiec it takes https://github.com/spree/spree/blob/master/api/app/serializers/spree/v2/storefront/image_serializer.rb
from the base,
any reason for this suggestion?

Reply:
@poudelprakash it should work, but we use a different class here: https://github.com/spree-contrib/spree_multi_vendor/blob/master/app/models/spree/vendor_image.rb

Reply:
I have now added serializer and spec for vendor_image, please check :)